api/app/schemas/morecast_v2.py

```diff
@@ -91,3 +103,22 @@ class ObservedDaily(BaseModel):
 class ObservedStationDailiesResponse(BaseModel):
     """ Yesterday station dailies response """
     dailies: List[ObservedDaily]
+
+
+class WeatherIndeterminate(BaseModel):
+    """ Used to represent a predicted or actual value """
+    station_code: int
+    station_name: str
+    determinate: WeatherDeterminate
+    utcTimestamp: datetime
```

Comment:
Should `utcTimestamp`be snake case?

Reply:
Yup it should, not sure why the linter doesn't catch that

Reply:
Fixed in: https://github.com/bcgov/wps/pull/2786/commits/128d1b9198f50d6f81a984bf58f3cb3c5a195ad8